dkist/dataset/tiled_dataset.py

```diff
@@ -184,6 +206,9 @@ def plot(self, slice_index, share_zscale=False, fig=None, **kwargs):
         fig : `matplotlib.figure.Figure`
             A figure to use for the plot. If not specified the current pyplot
             figure will be used, or a new one created.
+        limits_from_wcs : `bool`
```

Comment:
I'm not convinced `limits_from_wcs` is the best name for this option. Suggestions welcome!

Reply:
Perhaps `auto-rotate` or similar?

Reply:
Maybe `auto_orient`? I like where you're going, but I'm worried "rotate" might imply that it can rotate by something other than 180 degrees.

Reply:
Given the changes I suggest to the algorithm above, I think something like `auto_invert_axes` or something?